test/Spec.hs

```diff
@@ -1,2 +1 @@
-main :: IO ()
-main = putStrLn "Test suite not yet implemented"
+{-# OPTIONS_GHC -F -pgmF hspec-discover #-}
```

Comment:
This is just a trick to save you some boilerplate. Thanks to this hspec discovers all the tests modules without you having to manually create "root test file". Described here: https://hspec.github.io/hspec-discover.html